packages/gatsby-theme-carbon/src/templates/Default.js

```diff
@@ -9,10 +9,19 @@ import EditLink from '../components/EditLink';
 import NextPrevious from '../components/NextPrevious';
 import PageTabs from '../components/PageTabs';
 import Main from '../components/Main';
+import useMetadata from '../util/hooks/useMetadata';
 
 const Default = ({ pageContext, children, location, Title }) => {
   const { frontmatter = {}, relativePagePath, titleType } = pageContext;
-  const { tabs, title, theme, description, keywords } = frontmatter;
+  const {
+    tabs,
+    title,
+    theme: frontmatterTheme,
```

Comment:
Can you explain where `frontmatterTheme` comes from? Can we set theming in the frontmatter for pages?

Reply:
This was a recommendation @vpicone gave me. Let me see if I can summarize it. We wanted to need to rename the `theme` prop to `frontmatterTheme` (which is what that line is doing). Later, on line 52, we are assigning it's value || the value from `interiorTheme` to the theme const. This determines the color of the default pages. Vince, can you double check my math here ;)